sys/include/net/gnrc/tcp.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015 Simon Brummer
+ * Copyright (C) 2017 Simon Brummer
```

Comment:
AFAIK replacing the copyright year is not legit. Only updating it. You should change to either
```
Copyright (C) 2015, 2017 Simon Brummer
```
or
```
Copyright (C) 2015-2017 Simon Brummer
```

Btw. whether the copyright belongs to you or the university is yet another topic I don't want to discuss right here, right now. ;-)